Website/components/AboutPage.tsx

```diff
@@ -0,0 +1,343 @@
+                    src="/Gluon_Reverse_Blue2.png"
+                    alt={"GraalVM Logo"}
+                    width={100}
+                    height={100}
+                  ></Image>
+                </div>
+                <span className="text-gray-900 dark:text-white">GluonFX</span>
+              </div>
+            </div>
+          </div>
+          <div className="p-6 rounded-xl shadow-md">
+            <h3 className="text-sm font-medium mb-2 ">Website</h3>
+            <div className="flex flex-wrap items-center gap-3">
+              <div className="flex items-center space-x-2">
+                <div className="h-8 w-8 bg-gray-200 dark:bg-gray-800 rounded-lg flex items-center justify-center">
+                  <RiNextjsFill className="text-gray-800 dark:text-gray-200" />
+                </div>
+                <a
+                  href="https://nextjs.org/"
+                  className="text-lg font-medium hover:underline text-gray-900 dark:text-white"
+                  aria-label="View Apache 2.0 License"
+                  target="_blank"
+                >
+                  Next.js
+                </a>
+              </div>
+              <div className="flex items-center space-x-2">
+                <div className="h-8 w-8 bg-blue-100 dark:bg-blue-800 rounded-lg flex items-center justify-center">
+                  <RiTailwindCssFill className="text-blue-400 dark:text-blue-300" />
+                </div>
+                <a
+                  href="https://tailwindcss.com/"
+                  className="text-lg font-medium hover:underline text-gray-900 dark:text-white"
+                  aria-label="View Apache 2.0 License"
+                  target="_blank"
+                >
+                  tailwindcss
+                </a>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        {/* Project Owner Section */}
+        {repoData && (
+          <div className="flex flex-col md:flex-row items-stretch gap-4 w-full">
+            {/* Maintainer Card */}
+            <section className="p-8 rounded-2xl shadow-md flex-1 flex items-center min-h-[200px]">
+              <div className="flex items-center space-x-6 w-full">
+                <Image
+                  src={repoData.owner.avatar_url}
+                  alt={`${repoData.owner.login}'s profile picture`}
+                  className="w-24 h-24 rounded-full shadow-lg"
+                  width={96}
+                  height={96}
+                  loading="lazy"
+                />
+                <div>
+                  <h2 className="text-2xl font-bold">Saptarshi Sarkar</h2>
+                  <p className="text-lg mt-1">Maintainer</p>
+                  <a
+                    href={repoData.owner.html_url}
+                    className="inline-flex items-center hover:underline mt-2"
+                    aria-label="View GitHub profile"
+                  >
+                    <span>@{repoData.owner.login}</span>
```

Comment:
Can you make this part more responsive for mobile devices?
The text seems to come out of the black card.
![Screenshot From 2025-03-14 21-15-13](https://github.com/user-attachments/assets/8853c293-0429-4314-827e-090d0df83a3d)